pyproject.toml

```diff
@@ -1,3 +1,3 @@
 [build-system]
 # PEP 518 - minimum build system requirements
-requires = ["setuptools", "wheel", "Cython>=0.29", "packaging", "pip", "numpy", "mpi4py"]
+requires = ["setuptools", "wheel", "Cython>=0.29", "packaging", "pip", "numpy", "mpi4py", "versioneer"]
```

Comment:
We have to make sure to add the new dependency in our Spack package. 

Is it reasonable to add a new dependency just for the package version?

Reply:
Fixed in 2a28881.